### PR TITLE
Fix logging format for Mint (#194,#196 and #197)

### LINKED ISSUE #194: Functional test: function name should not contain the test name
Function name should not contain the test name. It should contain the API that was intended to be tested.

```
{"name":"Minio.Net","function":"BucketExists_Test","duration":817,"status":"PASS","alert":"","message":"","error":""}

```

`BucketExists_Test ` should not be the value of the function field. It should be something like `Task<bool> BucketExistsAsync(string bucketName, CancellationToken cancellationToken = default(CancellationToken)`

### PR DESCRIPTION
**Pull request by poornas**
Fixes #194, #196 and #197

### CODE REVIEW

File: Minio.Functional.Tests/MintLogger.cs [hunk 1]
```diff
@@ -72,12 +72,11 @@ internal class MintLogger
...
       // actual low level exception/error thrown by the program
       public string error {get; private set;}
-
-      public MintLogger(string function, string description,TestStatus status,System.TimeSpan duration,string alert = "",string message="", string error="", Dictionary<string,string> args=null)
+      public MintLogger(string testName,string function, string description,TestStatus status,System.TimeSpan duration, string alert = null,string message=null, string error=null, Dictionary<string,string> args=null)
       {
         this.function = function;
         this.duration = (int)duration.TotalMilliseconds;
-
+        this.name = this.name + ":" + testName;
```
Comment (kannappanr) [inline Minio.Functional.Tests/MintLogger.cs]: a space after the ":" will be more readable, if we have to look at it.

Comment (poornas) [reply Minio.Functional.Tests/MintLogger.cs]: sure.

File: Minio.Functional.Tests/FunctionalTest.cs [hunk 2]
```diff
@@ -1188,17 +1219,17 @@ private static void runQuickTests(MinioClient minioClient)
...
+            catch (Exception ex)
             {
-                new MintLogger("GetObject_Test3","Tests whether GetObject returns all the data",TestStatus.FAIL,(DateTime.Now - startTime),"",e.Message, e.ToString()).Log();
-                Assert.Fail();
+                    Assert.Fail();
+                    throw ex;
             }
             await TearDown(minio, bucketName);
-            new MintLogger("GetObject_Test3","Tests whether GetObject returns all the data",TestStatus.PASS,(DateTime.Now - startTime)).Log();
+            new MintLogger("GetObject_Test3",getObjectSignature2,"Tests whether GetObject returns all the data",TestStatus.PASS,(DateTime.Now - startTime)).Log();
```
Comment (kannappanr) [inline Minio.Functional.Tests/FunctionalTest.cs]: function field should have the function definition itself.

Comment (poornas) [reply Minio.Functional.Tests/FunctionalTest.cs]: getObjectSignature2 is just the function signature. Not sure what else is expected here?

**Comment by kannappanr** [comment]
Please take a look at the Mint log requirements at https://github.com/minio/mint/ 
Specifically at the `function` field. Here is a sample value `"getBucketLocation ( array $params = [] )"` . It doesn't say just 'getBucketLocation` but the whole function signature.

**Comment by poornas** [comment]
that's right, @kannappanr - I  have just defined a static string with function signature, and it will be passed to the log correctly.

**Pull request merged**